### PR TITLE
Odometry Merge I: The First One

### PR DESCRIPTION
**Pull request by 122004**
No actual odometry in this one, but it _does_ have a working drivetrain.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+# gradle/wrapper/gradle-wrapper.properties
+
+# # VS Code Specific Java Settings
+.classpath
+.project
+.settings/
+bin/
+imgui.ini
+
+
```
Comment (jjesse1972) [inline .gitignore:160]: update .gitignore to ignore all gradle downloaded artifacts. anything in .gradle directory

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+# End of https://www.gitignore.io/api/c++,java,linux,macos,gradle,windows,visualstudiocode
```
Comment (jjesse1972) [inline .gitignore:161]: update .gitignore to ignore build output anything in ./build directory

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+  /**
+   * This function is run when the robot is first started up and should be used for any
+   * initialization code.
+   */
+
+  
+  public static TestCommand command;
+  public static TestSubsystem subsystem;
+  // public NetworkTableEntry value_P;
```
Comment (jjesse1972) [inline src/main/java/frc/robot/Robot.java:30]: Remove this if your not going to use it. Dont clutter the code base with stuff that doesnt work.

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+  public void robotInit() {
+    RobotContainer.init();
+    OI.init();
+    subsystem = new TestSubsystem();
+    command = new TestCommand(subsystem);
+    System.out.println(command == null);
+    System.out.println(subsystem == null);
+    subsystem.register();
+    CommandScheduler.getInstance().setDefaultCommand(subsystem, command);
+    // Map<String,Object> map = new HashMap<String,Object>();
```
Comment (jjesse1972) [inline src/main/java/frc/robot/Robot.java]: Remove commented out shuffleboard stuff

File: src/main/java/frc/robot/Robot.java [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+   */
+  @Override
+  public void robotPeriodic() {
+    // Runs the Scheduler.  This is responsible for polling buttons, adding newly-scheduled
+    // commands, running already-scheduled commands, removing finished or interrupted commands,
+    // and running subsystem periodic() methods.  This must be called from the robot's periodic
+    // block in order for anything in the Command-based framework to work.
+    CommandScheduler.getInstance().run();
+    //System.out.println(value_P.getDouble(-1));
+    /*if (update.getBoolean(false)){
```
Comment (jjesse1972) [inline src/main/java/frc/robot/Robot.java]: Removed commented code

**Review by jjesse1972 — CHANGES_REQUESTED** [review]

**Review by jjesse1972 — APPROVED** [review]

**Review by BenWertz — APPROVED** [review]

**Pull request merged**